### ISSUE TITLE
Adding Dutch as a language option

### ISSUE THREAD

**Issue by MaartenBorsje** [MonitorControl/MonitorControl#502]
**Checklist**
- [ x] I have searched for existing issues

**Description**
I couldn't find a Dutch translation for this app. I do like to help translating. I already translated a Locolizedable.strings

**Comment by waydabber** [comment]
Hi @MaartenBorsje - Great, thank you! If you do the translation, please use the new 3.0.0 localizable files as a starting point (it is now being merged into the Master): https://github.com/MonitorControl/MonitorControl/pull/520

**Comment by waydabber** [comment]
Hi, fyi - 3.0.0 is out and is merged into Master. Please let us know if you'd like to work on this. Thanks!

**Comment by JoniVR** [comment]
I can honestly do dutch as well if we want it (primary language), I just haven't found much need for it so far, but if @MaartenBorsje wants to do it, fine by me too! :D

**Comment by MaartenBorsje** [comment]
@JoniVR @waydabber I'm happy to do it. It will help some of my folks. But I'm fairly new to GitHub. Where do you want me to drop the files?

**Comment by waydabber** [comment]
Hi @MaartenBorsje, If you do something like this, then it will be fine:

https://github.com/MonitorControl/MonitorControl/issues/528

**Comment by MaartenBorsje** [comment]
> Hi @MaartenBorsje, If you do something like this, then it will be fine:
> 
> #528

Oh I see. I noticed that in multiple languages some stuff isn't translated. Is that intentional?

**Comment by waydabber** [comment]
No, these are new texts that are still not translated. You should translate all English content to the target language.

**Comment by MaartenBorsje** [comment]
[nl.lproj.zip](https://github.com/MonitorControl/MonitorControl/files/7027875/nl.lproj.zip)

Here is the translation

**Comment by JoniVR** [comment]
Hi @MaartenBorsje! 

You seem to have missed some files? Or was this intentional? 
I've extracted a zip with english files if you want to do all of them:
[nl.zip](https://github.com/MonitorControl/MonitorControl/files/7033911/nl.zip)

**Comment by MaartenBorsje** [comment]
[nl.zip](https://github.com/MonitorControl/MonitorControl/files/7037945/nl.zip)
That wasn't intentional. I just didn't know there were more. But here they are.
